cmd/inmap/configExampleGEOSChem.toml

```diff
@@ -1,6 +1,6 @@
-InMAPData= "${GOPATH}/src/github.com/spatialmodel/inmap/cmd/inmap/testdata/preproc/inmapData_GEOSChem.ncf"
+InMAPData= "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/preproc/inmapData_GEOSChem.ncf"
```

Comment:
In order for these paths to point to the right files, these should all be of the form `inmap/cmd/inmap` rather than `inmap/inmap`